addon/-private/modifier-native.js

```diff
@@ -23,4 +24,17 @@ class Modifier {
 
 Ember._setModifierManager(createManager, Modifier);
 
+export function modifier(modifierFn) {
+  assert(
+    'You must pass a function as the first argument to the `modifier` function',
+    modifierFn !== undefined && typeof modifierFn === 'function'
+  );
+
+  return class extends Modifier {
+    didReceiveArguments(positional, named) {
+      modifierFn(this.element, positional, named);
```

Comment:
Ahh, now it makes sense. I'd like to see this documented in the README. Could that be ok?

Reply:
I've updated the documentation, you can take a look.